learn/tanssi/services-payment.md

```diff
@@ -0,0 +1,45 @@
+
+All this operation carries costs that must be covered by the appchains that want to obtain the services. This article covers the general aspects of those costs and associated service payments.
+
+## Costs to Cover  {: #costs-to-cover }
+
+There are three main costs associated with block production as a service that any appchain must cover using Tanssi tokens to deploy successfully and get the block production services:
+
+- **Registration deposit** - is the initial deposit that is locked from the account that signs the appchain registration transaction
+- **Block producers assignment** - every time the Tanssi protocol assigns block producers, which happens once per session, a fixed fee is charged. This discourages a faulty appchain (whose runtime logic fails to produce valid transitions and, therefore, a valid block) from requesting being assigned nodes that will not be able to produce valid blocks
+- **Block production** - appchains need to pay for each block that is produced on their behalf. Since the protocol selects and assigns the block producers on a per-session basis, appchains must have enough funds to cover all the blocks to be produced in an entire session to be served
+
+On Dancebox, the Tanssi testnet, the current configuration is the following:
+
+=== "Dancebox"
+    |         Variable          |                                   Value                                   |
+    |:-------------------------:|:-------------------------------------------------------------------------:|
+    |   Registration deposit    |         {{ networks.dancebox.costs.registration_deposit }} DANCE          |
+    | Block producer assignment | {{ networks.dancebox.costs.cost_per_assignment }} microDANCE per session |
+    |     Block production      |    {{ networks.dancebox.costs.cost_per_block }} microDANCE per block     |
+
+To ensure block production in the next session, the total balance must be at least enough to cover the block producer assignment cost plus the cost to produce the {{ networks.dancebox.session.blocks }} blocks that comprise an entire session.
+
+!!! note
+    Although these costs are currently fixed, as protocol development progresses, they might become dynamic, varying in response to the network's workload.
+
+## Tipping {: #tipping }
+
+Besides the regular costs presented in the previous section, there is an additional and optional fee: the tipping mechanism. Appchains are free to offer a tip to be served with priority over the rest, which, in times when the network is experiencing a high demand that can not be met with the available resources, could be the difference between producing blocks or stalling for an entire session at least.  
```

Comment:
I would switch this paragraph a bit so that we state "The Problem -> The Solution"

So you start like:

"On some occasions, Tanssi might experience high demand for its block production services that can not be met with the available resources. For example, if for the next session, there are ten appchains and Tanssi can only serve eight, two appchains might stall for the entire session duration...."